decompiler/structures/pseudo/expressions.py

```diff
@@ -277,6 +277,18 @@ def copy(self) -> Symbol:
         return Symbol(self.name, self.value, self._type.copy(), self.tags)
 
 
+class StringSymbol(Symbol):
+    """Represents a global string constant (const char[size])"""
+    def __str__(self):
+        return f'"{self._name}"'
+
+    def __repr__(self):
+        return f"string {self.name} at {self.value}"
+
+    def copy(self) -> StringSymbol:
+        return StringSymbol(self.name, self.value, self._type.copy(), self.tags)
+
```

Comment:
We need a unittest with the problem we had before, e.g. `if (value == "string")`

Reply:
Added a small function + test in the `globals` binary.